README.md

```diff
@@ -50,11 +50,19 @@ Table of Content
 
 # Infrastructure
 
+## Client Libraries
+- [Go Tezos](https://github.com/DefinitelyNotAGoat/go-tezos)
+- [eztz] https://github.com/TezTech/eztz
```

Comment:
Please put the url in between parentheses to render it as a hyperlink.
`- [eztz](https://github.com/TezTech/eztz)`